job/job_test.go

```diff
@@ -194,7 +194,7 @@ func TestHasActiveWorkflow(t *testing.T) {
 			},
 			status: true,
 		},
-		{name: "test inactive workflow",
+		{name: "test running workflow",
 			wcl: &tw.WorkflowContextList{
 				WorkflowContexts: []*tw.WorkflowContext{
 					&tw.WorkflowContext{
```

Comment:
maybe change the workflow ids from `active-fake-workflow-blah` to `pending-fake...`, `running-fake...`, etc. so there's less room for confusion?

Reply:
good call, I forgot